diff --git a/src/rules.ruleset b/src/rules.ruleset
new file mode 100644
index 0000000..9cbfc7c
--- /dev/null
+++ b/src/rules.ruleset
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Hack F5" Description=" " ToolsVersion="10.0">
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <Rule Id="SA1005" Action="None" />
+    <Rule Id="SA1515" Action="None" />
+    <Rule Id="SA1009" Action="None" />
+    <Rule Id="SA1028" Action="None" />
+    <Rule Id="SA1101" Action="None" />
+    <Rule Id="SA1310" Action="None" />
+    <Rule Id="SA1407" Action="None" />
+    <Rule Id="SA1512" Action="None" />
+    <Rule Id="SA1600" Action="None" />
+    <Rule Id="SA1601" Action="None" />
+    <Rule Id="SA1602" Action="None" />
+    <Rule Id="SA1633" Action="None" />
+    <Rule Id="SA0001" Action="None" />
+    <Rule Id="SA1404" Action="None" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.CodeQuality.Analyzers" RuleNamespace="Microsoft.CodeQuality.Analyzers">
+    <Rule Id="CA1001" Action="None" />
+    <Rule Id="CA1062" Action="None" />
+    <Rule Id="CA1066" Action="None" />
+    <Rule Id="CA1707" Action="Info" />
+    <Rule Id="CA1720" Action="Info" />
+    <Rule Id="CA1815" Action="Info" />
+    <Rule Id="CA1822" Action="Info" />
+    <Rule Id="CA2007" Action="None" />
+    <Rule Id="CA1030" Action="None" />
+    <Rule Id="CA9998" Action="None" />
+  </Rules>
+</RuleSet>
